Strategy/FileFormReader.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherMonitoringSystem.model;
+
+namespace WeatherMonitoringSystem.Strategy
```

Comment:
the namespace should indicate the purpose of the module, not the implementation.
maybe something like:
```suggestion
namespace WeatherMonitoringSystem.Serializers
```